gradle/js.gradle

```diff
@@ -45,10 +45,6 @@ kotlin {
     sourceSets {
         jsMain.dependencies {
             api "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlin_version"
-
-            api group: 'org.jetbrains.kotlinx', name: 'kotlinx-coroutines-core-js', version: coroutines_version
```

Comment:
Don't we need to change it to `org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version` or is it configured somewhere in a common module?

Reply:
It's in common